sdk/servicebus/azure-servicebus/tests/test_sb_client.py

```diff
@@ -108,7 +108,7 @@ def test_sb_client_writeonly_credentials(self, servicebus_authorization_rule_con
             with client.get_queue_sender(servicebus_queue.name) as sender:
                 sender.send(Message("test"))
 
-                with pytest.raises(ServiceBusError): 
+                with pytest.raises(TypeError):
```

Comment:
I'm somewhat surprised that this is the only location ServiceBusError would change in tests.  Let's make sure we have a full test run.

Reply:
Locally all test cases succeeded. I think this one + the async counterpart are the only two occurrences.